### DISCUSSION TITLE
Subscription Link Page

### DISCUSSION THREAD

**Discussion by iamtheted** [Gozargah/Marzban#344] [Ideas]
سلام و درود به عزیزان مرزبان💙
پیج لینک ساب رو [محمد](https://t.me/MuhammadAshouri) جان نوشته.
فقط یه تغییراتی توی رنگ‌ و لعاب و فونت داده شده.
سازگار با نسخه Latest 0.3.2 و نسخه Dev مرزبان.

[sub-page-light.zip](https://github.com/Gozargah/Marzban/files/11929861/sub-page-light.zip)
[sub-page-dark.zip](https://github.com/Gozargah/Marzban/files/11929875/sub-page-dark.zip)

قدم اول:
```
cd /opt/marzban
```
 اگر تم لایت رو می‌خواین:
```
wget https://github.com/Gozargah/Marzban/files/11929861/sub-page-light.zip && unzip sub-page-light.zip
```
اگر تم دارک رو می‌خواین:
```
wget https://github.com/Gozargah/Marzban/files/11929875/sub-page-dark.zip && unzip sub-page-dark.zip
```
قدم بعدی: 
```
 nano docker-compose.yml 
``` 
برای آدرس دادن در docker-compose.yml این سه خط را به بخش volume اضافه کنید:
دقت کنید که این سه خط باید در یک امتداد با بقیه خطای بخش volume باشند.
```
 - /opt/marzban/sub-page/config.py:/code/config.py
 - /opt/marzban/sub-page/subscription.py:/code/app/views/subscription.py
 - /opt/marzban/sub-page/subscription:/code/app/templates/subscription
```

قدم آخر: 
```
marzban restart
```
------------------------------------------------------------------------------------ 
Light Theme

![Light-1](https://github.com/Gozargah/Marzban/assets/135175623/af14aa90-db4e-4fc8-a555-aca4699239d3)

![Light-2](https://github.com/Gozargah/Marzban/assets/135175623/1633821d-cba5-4008-b288-e045d00b7164)

![Light-3](https://github.com/Gozargah/Marzban/assets/135175623/4c5b628c-195c-44c5-9965-bd8af63cf356)
------------------------------------------------------------------------------------ 
Dark Theme

![Dark-1](https://github.com/Gozargah/Marzban/assets/135175623/07a081a0-4040-4d66-8d62-bf1fd88fbdc3)

![Dark-2](https://github.com/Gozargah/Marzban/assets/135175623/cc84991a-9944-4111-9f85-b77e9034202e)

![Dark-3](https://github.com/Gozargah/Marzban/assets/135175623/15751ae4-ad2d-4f94-8465-6992f73ef646)
------------------------------------------------------------------------------------ 

Original Author: [Mohammad Ashouri](https://t.me/MuhammadAshouri)
Modification Contributors: [TED](https://t.me/iamtheted), [Mohsen](https://t.me/Feedacle), [Error](https://t.me/iERROR_XD)

**Comment by JavadWolf-af** [comment]
سلام تد
کاربرانی که از قبل پنل داشتن و اپدیت کردن پوشه مرزبان در /opt/marzban/ نیست بلکه در root/marzban هست و من الان به مشکل خوردم صفحه برای من کاملا بهم خورده است من حتی به جای opt از root استفاده کردم ولی صفحه بهم ریخته و دیتابیس کار نمیکند ...

**Comment by iamtheted** [comment]
سلام عزیزم
امیدوارم حالت خوب باشه.
خب این رو خودم رو پنل جدید مرزبان امتحان کردم که مسیرش opt/marzban/ هست.
شما هم با نسخه جدید مرزبان امتحان کنین اوکی میشه. ✅

**Reply by smaghili** [reply]
سلام . وقتتون بخیر. 
بله وقتی لینک ساب روی  پنل اصلی باشه جواب میده .ولی وقتی روی هاست ایرانی باشه جواب نمیده . و صفحه خراب نشون داده میشه .

**Reply by iamtheted** [reply]
خب اول از همه ی پوشه به اسم static جفت پوشه sub میسازید.

![photo_2023-06-25_02-40-43](https://github.com/Gozargah/Marzban/assets/135175623/9eb15e29-3a41-4866-a451-0561f4800481)
---------------------
بعد تمام فایل های توی پوشه subscription جز فایل index.html رو می ریزید توی پوشه static و تمام.

![photo_2023-06-25_02-43-21](https://github.com/Gozargah/Marzban/assets/135175623/7a9b3486-34fc-4f42-b094-d3d21f5c1246)
---------------------
در خصوص پیج لینک ساب از طریق هاست ایران اگر سوال دیگری دارید به آیدی یا بات زیر پیام بدین.

[AC Lover](https://t.me/AC_Lover)
[AC Lover Bot](https://t.me/AC_LoverBot)

**Comment by navabmis** [comment]
درود 
چطور این صفحه فارسی میشه

**Reply by iamtheted** [reply]
سلام عزیز 
امیدوارم حالت خوب باشه. 💙
این [لینک](https://github.com/Gozargah/Marzban/pull/338) رو در خصوص پیج ساب فارسی ببینید.

**Comment by t-51** [comment]
ببخشید میشه عکس پروفایل و عوض کرد ؟

**Comment by iamtheted** [comment]
> ببخشید میشه عکس پروفایل و عوض کرد ؟

سلام 
آره میشه منتها باید توی فایلا بگردین ببینین کدوم بخشش مختص اون هست تا بتونین عوضش کنین.

**Comment by ituser96** [comment]
سلام. وقتی این کانفیگ رو انجام بدیم، لینک این صفحه کجا هست؟ از کجا باید بیاریمش؟

**Reply by iamtheted** [reply]
سلام عزیز
کافیه لینک Sub رو بزنید داخل بروزر تا این صفحه بالا بیاد.
هرچند این ورژن قدیمی پیج ساب هست.
ورژن جدیدش رو این زیر براتون میزارم.
https://github.com/MuhammadAshouri/marzban-templates
